### ISSUE TITLE
make check fails on Arch Linux

### ISSUE THREAD

**Issue by mxmilkb** [tim-janik/beast#130]
```
==> Starting check()...
  MODE     production
Traceback (most recent call last):
  File "<string>", line 1, in <module>
ModuleNotFoundError: No module named 'md5'
  RUN      sfi-check
  TESTING  Sfidl factory
  TESTING  Testing Option parser
  TESTING  Testing CodeGenerator::rename()
  CHECK    aidacc/tests/check--t301-pass.idl
  CHECK    aidacc/tests/check--t302-fail.idl
  CHECK    aidacc/tests/check--t304-cxxserver-output
  CHECK    aidacc/tests/check--t305-idlcode-compile
  RUN      aidacc/build-test
  RUN      out/bse/t279-assertions-test
  PASS     out/bse/t279-assertions-test
  GEN      out/tests/bsefiles.lst
  CHECK    out/tests/bsefiles.lst-a
/usr/lib/ladspa/lsp-plugins-ladspa.so: Bse LADSPA plugin registration failed: /usr/lib/ladspa/lsp-plugins-ladspa.so: undefined symbol: _ZN3lsp17builtin_resourcesE
fluidsynth: error: SDL2 not initialized
./media/Effects/atan-canyon.bse:6: invalid date specification (0 < 631152000000000, gmt-diff=-3600000000) in date: "2013-03-24 16:48:33"
./media/Effects/atan-canyon.bse:7: invalid date specification (0 < 631152000000000, gmt-diff=-3600000000) in date: "2003-10-25 21:02:35"
make: *** [tests/Makefile.mk:85: out/tests/bsefiles.lst-a-test] Error 1
==> ERROR: A failure occurred in check().
```

**Comment by tim-janik** [comment]
Hm, I *thought* our tests didn't try to load LADSPA plugins, but maybe that's not disabled...?
About the fluidsynth error, @swesterfeld do you have an idea?

**Comment by mxmilkb** [comment]
I didn't notice this before when running from the build directory, but running `beast` after installing gives me this output;
```
% beast
/usr/lib/ladspa/lsp-plugins-ladspa.so: Bse LADSPA plugin registration failed: /usr/lib/ladspa/lsp-plugins-ladspa.so: undefined symbol: _ZN3lsp17builtin_resourcesE
fluidsynth: error: SDL2 not initialized
```
Things the main window opens fine.

**Comment by sadko4u** [comment]
This problem has been fixed in the 1.1.11 relase. Related issue:
https://github.com/sadko4u/lsp-plugins/issues/58

**Comment by mxmilkb** [comment]
Thanks :)
